apps/antalmanac/src/lib/download.ts

```diff
@@ -281,45 +283,42 @@ export function getEventsFromCourses(events = AppStore.getEventsInCalendar()): E
                     if (location.days === undefined) return null;
```

Comment:
This should be enforced by the linter, so this change is purely personal preference. After reading through lots of code, I really like it when things are consistent, even if they're marginally more verbose. I get throw off sometimes when reading through a bunch of if-statements with newlines and curly braces and then coming across one if-statement that's compacted into one line.

```suggestion
                    if (location.days === undefined) {
                      return null;
                    }
```